app/src/main/java/ch/epfl/sdp/game/Game.java

```diff
@@ -32,11 +35,20 @@ public static Game getInstance() {
      * change the map before launching)
      */
     private Game() {
+        mapApi = null; // is it okay?
```

Comment:
I think it is, so you can remove the comment. The only other way is to have a temporary api that does nothing.